R/mod_uploadPackage.R

```diff
@@ -251,7 +251,7 @@ uploadPackageServer <- function(id, user) {
             
             if (grepl("^[[:alpha:]][[:alnum:].]*[[:alnum:]]$", uploaded_packages$package[i])) {
               # run pkg_ref() to get pkg version and source info
-              ref <- riskmetric::pkg_ref(uploaded_packages$package[i])
+              ref <- riskmetric::pkg_ref(uploaded_packages$package[i], repos = c("https://cran.rstudio.com"))
```

Comment:
I would suggest:

```r
ref <- riskmetric::pkg_ref(uploaded_packages$package[i], source = "pkg_cran_remote", repos = c("https://cran.rstudio.com"))
```

So you can also patch #412 and #385

Reply:
Resolved with [32e8074](https://github.com/pharmaR/riskassessment/pull/413/commits/32e807439ca13f553d942d1e729014c4d4a1bdaf)